3.train_ebimage/1.random_forest/random-forest.py

```diff
@@ -0,0 +1,41 @@
+# %% Initialization
+import sys
+from pathlib import Path
+
+basefolder_loc = Path(__file__).parents[2]
+
+import os
+import numpy as np
+import pandas as pd
+
+from sklearn.model_selection import train_test_split
+
+sys.path.append(str(basefolder_loc))
+from utils import create_metrics
+
+data_dir = os.path.join(basefolder_loc, "2.process-data", "data")
+
+
+def load_data(file_name="train_processed.tsv.gz"):
```

Comment:
Perhaps [this comment is out-of-scope of the current PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#opening-an-issue-for-an-out-of-scope-suggestion) but we should consider splitting this function out of the `random-forest.py` file.

It will help massively with [DRY principles](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself). 

It will be used a bunch of times by many other scripts. It would be great if it could be imported. I add #30 to discuss this - feel free to address in this PR or we can save it for later

Reply:
Good point, but indeed better outside the scope of the this pr